### DISCUSSION TITLE
Misleading "supported artifacts" error message

### DISCUSSION THREAD

**Discussion by stdedos** [community/community#129909] [Projects and Issues]
### Select Topic Area

Bug

### Feature Area

Issues

### Body

I am trying to upload an `archive.tgz`, and I receive this message:

![image](https://github.com/community/community/assets/133706/8a65e1a7-0d58-47c3-bab6-c70b0a1f580d)

... which makes no sense. You support (theoretically) `tgz`, but you **demand** that it is `tar.gz`.

### Guidelines

- [X] I have read the above statement and can confirm my post is relevant to the GitHub feature areas [Issues](https://docs.github.com/en/issues/tracking-your-work-with-issues/about-issues) and/or [Projects](https://docs.github.com/en/issues/planning-and-tracking-with-projects/learning-about-projects/about-projects).

**Comment by danisoaresl** [comment]
Hello, have you tried contacting the github team?

**Reply by stdedos** [reply]
... and the GH team will send me "somewhere here" 😒 

Why bother?
